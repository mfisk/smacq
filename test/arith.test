#!/bin/sh

bin/smacqq -O -m <<EOF 2>/dev/null
print -v, seq, ipid, seq.double, ipid.uint32 from pcapfile($TEST/0325@1112-snort.pcap) where seq.double > ( ipid.uint32 * 10000000 )
EOF
