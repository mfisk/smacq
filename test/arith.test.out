seq = 3544678043	ipid = 325	seq.double = 3.54468e+09	ipid.uint32 = 325
seq = 3544678082	ipid = 326	seq.double = 3.54468e+09	ipid.uint32 = 326
seq = 3544678125	ipid = 327	seq.double = 3.54468e+09	ipid.uint32 = 327
seq = 3544678126	ipid = 328	seq.double = 3.54468e+09	ipid.uint32 = 328
seq = 3593394780	ipid = 329	seq.double = 3.59339e+09	ipid.uint32 = 329
seq = 3593444673	ipid = 330	seq.double = 3.59344e+09	ipid.uint32 = 330
seq = 3593444674	ipid = 331	seq.double = 3.59344e+09	ipid.uint32 = 331
seq = 3593444674	ipid = 332	seq.double = 3.59344e+09	ipid.uint32 = 332
seq = 3593444746	ipid = 333	seq.double = 3.59344e+09	ipid.uint32 = 333
seq = 3593444883	ipid = 334	seq.double = 3.59344e+09	ipid.uint32 = 334
seq = 3593509274	ipid = 335	seq.double = 3.59351e+09	ipid.uint32 = 335
seq = 3593571492	ipid = 336	seq.double = 3.59357e+09	ipid.uint32 = 336
seq = 3593649239	ipid = 337	seq.double = 3.59365e+09	ipid.uint32 = 337
seq = 3593700394	ipid = 338	seq.double = 3.5937e+09	ipid.uint32 = 338
seq = 3593755013	ipid = 339	seq.double = 3.59376e+09	ipid.uint32 = 339
seq = 3593800672	ipid = 340	seq.double = 3.5938e+09	ipid.uint32 = 340
seq = 3593845018	ipid = 341	seq.double = 3.59385e+09	ipid.uint32 = 341
seq = 3593509274	ipid = 342	seq.double = 3.59351e+09	ipid.uint32 = 342
seq = 3593571492	ipid = 343	seq.double = 3.59357e+09	ipid.uint32 = 343
seq = 3593755013	ipid = 344	seq.double = 3.59376e+09	ipid.uint32 = 344
seq = 3593800672	ipid = 345	seq.double = 3.5938e+09	ipid.uint32 = 345
seq = 3593845018	ipid = 346	seq.double = 3.59385e+09	ipid.uint32 = 346
seq = 3593649239	ipid = 347	seq.double = 3.59365e+09	ipid.uint32 = 347
seq = 3593700394	ipid = 348	seq.double = 3.5937e+09	ipid.uint32 = 348
seq = 3593509274	ipid = 349	seq.double = 3.59351e+09	ipid.uint32 = 349
seq = 3593571492	ipid = 350	seq.double = 3.59357e+09	ipid.uint32 = 350
seq = 3593755013	ipid = 351	seq.double = 3.59376e+09	ipid.uint32 = 351
seq = 3593800672	ipid = 352	seq.double = 3.5938e+09	ipid.uint32 = 352
seq = 3593845018	ipid = 353	seq.double = 3.59385e+09	ipid.uint32 = 353
seq = 3593649239	ipid = 354	seq.double = 3.59365e+09	ipid.uint32 = 354
seq = 3593700394	ipid = 355	seq.double = 3.5937e+09	ipid.uint32 = 355
seq = 3593445175	ipid = 356	seq.double = 3.59345e+09	ipid.uint32 = 356
seq = 3593445176	ipid = 357	seq.double = 3.59345e+09	ipid.uint32 = 357
seq = 3594686715	ipid = 358	seq.double = 3.59469e+09	ipid.uint32 = 358
seq = 3594727176	ipid = 359	seq.double = 3.59473e+09	ipid.uint32 = 359
